### PR TITLE
Add the TTF14 style guide

### PR DESCRIPTION
**Pull request by kabeaty**
#### What's the plan?

This PR adds in a style guide for the TTF14 site.
#### What's this PR do?

Adds a TTF14 style guide HTML page and styles, particularly TTF14festivalstyles.html.

Note: When this PR and the other that switches all styles to Sass are both merged in, then festival.css won't need to be checked in. It's in here for testing purposes to ensure everything looks good before they're merged together.
#### How should this be manually tested?

Run locally with:
- `grunt` to build css
- `make serve`

Navigate to the TTF14 styleguide and click around. Read through for completeness of content and the best wording of copy. Check that everything looks good. Try out the Example Code by clicking to toggle it to display and not display.
#### TODO
- [x] Check in TTF14 css
#### Smells?

Share any you see

### CODE REVIEW

**Comment by crccheck** [comment]
the images/ directory is getting cluttered. can you add a directory with the same name, "TTF14festivalstyles", and move the images there?

**Comment by crccheck** [comment]
well... that can be saved for a later PR too where we just add consistent namespaces for everything (including the sass). let's wait for one more styleguide.

File: TTF14festivalstyles.html [hunk 1]
```diff
@@ -0,0 +1,444 @@
+---
+layout: default
+title: Texas Tribune Festival 2014
```
Comment (crccheck) [inline TTF14festivalstyles.html]: title should be `The Texas Tribune Festival 2014`. But I think it's already looking long and it's looking compressed. "Festival 2014" should be enough.

screen shot 2014-08-18 at 11 16 06 am

**Comment by crccheck** [comment]
the neat hover effect in the main nav is gone now, namespaced to .tribtalk-page. was that intentional?

**Comment by kabeaty** [comment]
OK, sounds good on the images. In the other PR that changes everything over to Sass, I started namespacing the Sass with directories named 'festival' (I'll update this to 2014 to allow for other years to be added), 'styleguide' (for base style guide styles and those specific to each individual style guide but not included in the actual project), and 'tribtalk.' 

I'm good with changing it to 'Festival 2014'. In my notes on next things to do for style guide, I also have written down to change the styling for the homepage so the main nav list elements aren't looking so compressed, but I have a few things I'd like to do to improve the homepage and am thinking those could all go together.

I changed the hover colors for the side navs for individual style guides to match the colors used by festival and tribtalk. I agree there should still be a hover effect on the main page, too - I'll add that to the 'sassify-all' PR since styles are already switched over to Sass there if that works.

**Comment by crccheck** [comment]
+1

**Pull request merged**